common/vector_databases/lasr_vector_databases_faiss/.gitignore

```
@@ -0,0 +1 @@
+data/**
```

This may be slightly picky, but can we instead have a `data` directory, that is stored in the repository, with a `.gitkeep` and change the `.gitignore` to this?
```suggestion
data/**
!data/.gitkeep
```